arcgis-ios-sdk-samples/Display information/Read symbols from a mobile style/ReadSymbolsFromMobileStyleSymbolSettingsViewController.swift

```diff
@@ -120,17 +120,20 @@ class ReadSymbolsFromMobileStyleSymbolSettingsViewController: UITableViewControl
         } else if imageOperations[indexPath] != nil {
             return nil
         } else {
-            let operation = searchResult.symbol?.createSwatch(withWidth: 40, height: 40, screen: .main, backgroundColor: nil) { [weak self] (image, error) in
+            searchResult.symbol { [weak self] symbol, error in
```

Comment:
This error hasn't been handled, has it?

Reply:
Nope. 😳  
I assume the previous API will give nil when an error happens, and it doesn't proceed with a nil. So I just ignored the error in the new API. Should it be handled?

Reply:
I think so

Reply:
Fixed.